.github/.cspell/dictionary.txt

```diff
@@ -0,0 +1,6 @@
+accs
```

Comment:
acc means acceleration, and accs means accelerations.
If possible (and reasonable), I want to change the code not adding the dictionary.

Reply:
Replaced with "accelerations"